Changelog.md

```diff
@@ -1,5 +1,8 @@
 # FOSSA CLI Changelog
 
+## v3.7.12
```

Comment:
Should we pair this PR with first-party license scanning and make the release 3.8? The deprecation of monorepo (breaking change) and a significant new feature seem like a good reason to bump the minor version.

@spatten

Reply:
That works for me, as long as we're fine with waiting until the first-party license scanning is done

Reply:
@spatten please just ping me (or feel free to just resolve conflicts & merge this PR) whenever that is done then!

Reply:
As discussed in slack, merging after all to make #1203 simpler.